pandas/core/groupby.py

```diff
@@ -2608,6 +2608,8 @@ def _cython_agg_blocks(self, how, numeric_only=True):
         for block in data.blocks:
 
             values = block._try_operate(block.values)
```

Comment:
Ok,
Then I assume I would do the `._try_operate` after checking `is_datetime_or_timedelta_dtype` not to lose dtype.
But `._try_operate` on `DatetimeBlock` is just `values.view('i8')`
However the code just after `is_datetime_or_timedelta_dtype` is `values = values.view('int64')`
I assume both are not needed so I can remove `._try_operate`
Hope this doesn't destroy something else.